spec/suites/dom/DomUtilSpec.js

```diff
@@ -53,6 +53,31 @@ describe('DomUtil', function () {
 		});
 	});
 
+	describe('getClass', function () {
+		it('should return the element\'s class', function () {
+			el.className = 'bar foo baz';
+			expect(L.DomUtil.getClass(el)).to.eql('bar foo baz');
+		});
+		it('should return empty string if no className', function () {
+			el.id = 'testId';
+			expect(L.DomUtil.getClass(el)).to.eql('');
+		});
+	});
+
+	describe('setClass', function () {
+		it('should set className', function () {
+			el.className = '';
+			L.DomUtil.setClass(el, 'bar');
+			expect(el.className).to.eql('bar');
+		});
+
+		it('should set classNames', function () {

```

Comment:
Also here it would be preferred if 'should' is omitted from the description. Maybe also we can keep these descriptions consistent with the other one you added.
```suggestion
		it('sets the element\'s class', function () {

```